components/admin_console/s3_connection_test/index.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {testS3Connection} from 'mattermost-redux/actions/admin';
+
+import S3ConnectionTest from './s3_connection_test.jsx';
+
+export default connect(null, mapDispatchToProps)(S3ConnectionTest);
+
+function mapDispatchToProps(dispatch) {
+    return {
+        actions: bindActionCreators({
+            testS3Connection
+        }, dispatch)
+    };
+}
+
```

Comment:
Should add below line so that `props.actions.testS3Connection` becomes available at `S3ConnectionTest` component:
```
export default connect(null, mapDispatchToProps)(S3ConnectionTest);
```

Reply:
Oppss sorry, I was wrong. I see it at line 10 just now. I'm used to seeing that usually at the end of the file.

Reply:
😄